articles/TOC.yml

```diff
@@ -46,15 +44,23 @@
     href: language/toc.yml
   - name: Quantum simulators and resource estimators
     href: machines/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Quantum computing concepts
-  href: concepts/toc.yml
-- name: Further reading
-  href: for-more-info.md
-- name: Contributing to the QDK
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: QDK Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
+- name: Concepts 
+  items: 
+  - name: Quantum computing concepts
+    href: concepts/toc.yml
+  - name: Q# libraries
+    href: libraries/toc.yml
+- name: Reference 
```

Comment:
Looking at the rendered TOC, this creates an empty "Reference" node that may be easily confused with the "API Reference" node lower down in the TOC. If possible, it would be good to fix that before going live.

![image](https://user-images.githubusercontent.com/31516/80990889-fb3aff00-8deb-11ea-8e61-74e532d3dc27.png)

Reply:
Good call. We'll remove it.